rllib/algorithms/maml/maml.py

```diff
@@ -18,70 +17,159 @@
 from ray.rllib.execution.metric_ops import CollectMetrics
 from ray.rllib.evaluation.metrics import collect_metrics
 from ray.rllib.utils.annotations import override
-from ray.rllib.utils.deprecation import DEPRECATED_VALUE
+from ray.rllib.utils.deprecation import Deprecated, DEPRECATED_VALUE
 from ray.rllib.utils.metrics.learner_info import LEARNER_INFO
+from ray.rllib.utils.sgd import standardized
 from ray.rllib.utils.typing import TrainerConfigDict
 from ray.util.iter import from_actors, LocalIterator
 
 logger = logging.getLogger(__name__)
 
-# fmt: off
-# __sphinx_doc_begin__
-DEFAULT_CONFIG = with_common_config({
-    # If true, use the Generalized Advantage Estimator (GAE)
-    # with a value function, see https://arxiv.org/pdf/1506.02438.pdf.
-    "use_gae": True,
-    # GAE(lambda) parameter
-    "lambda": 1.0,
-    # Initial coefficient for KL divergence
-    "kl_coeff": 0.0005,
-    # Size of batches collected from each worker
-    "rollout_fragment_length": 200,
-    # Do create an actual env on the local worker (worker-idx=0).
-    "create_env_on_driver": True,
-    # Stepsize of SGD
-    "lr": 1e-3,
-    "model": {
+
+class MAMLConfig(TrainerConfig):
+    """Defines a configuration class from which a MAMLTrainer can be built.
+
+    Example:
+        >>> from ray.rllib.algorithms.maml import MAMLConfig
+        >>> config = MAMLConfig().training(use_gae=False).resources(num_gpus=1)
+        >>> print(config.to_dict())
+        >>> # Build a Trainer object from the config and run 1 training iteration.
+        >>> trainer = config.build(env="CartPole-v1")
+        >>> trainer.train()
+
+    Example:
+        >>> from ray.rllib.algorithms.maml import MAMLConfig
+        >>> from ray import tune
+        >>> config = MAMLConfig()
+        >>> # Print out some default values.
+        >>> print(config.lr)
+        ... 0.0004
```

Comment:
ellipses are for continuation of a previous line. I think you need to remove them (and obv. keep the indent) for this to be doc-tested correctly.

Not sure why this doesn't fail, do we turn `doctest` off somewhere for these examples? Last time I checked it was still part of the docs build.